src/main/java/io/cdap/delta/bigquery/BigQueryTarget.java

```diff
@@ -138,17 +138,15 @@ public void initialize(DeltaTargetContext context) throws Exception {
             });
     try {
       long maximumExistingSequenceNumber = Failsafe.with(retryPolicy).get(() ->
-              BigQueryUtils.getMaximumExistingSequenceNumber(context.getAllTables(), project, conf.getDatasetName(),
-                      bigQuery, encryptionConfig, MAX_TABLES_PER_QUERY));
+              BigQueryUtils.getMaximumExistingSequenceNumber(context.getAllTables(), conf.getDatasetProject(),
+                conf.getDatasetName(), bigQuery, encryptionConfig, MAX_TABLES_PER_QUERY));
```

Comment:
nit: revert ident change

Reply:
This project and conf.getDatasetproject() both are different project Id's.
Earlier it was pointing to project, now we are passing DatasetProjectId.

Reply:
I meant to say that we should revert the whitespace change (indent), not code change to point to the right project id